services/authentication/resources/roles_updates.py

```diff
@@ -0,0 +1,85 @@
+from daos.roles_updates_dao import RolesUpdatesDAO
+from daos.roles_dao import RolesDAO
+from db import Session
+from datetime import datetime
+from flask import jsonify
+
+class RolesUpdates:
+    @staticmethod
+    def add_admin_role(uuid):
+        session = Session()
+        role_update = RolesUpdatesDAO(id = int(uuid)*99999-1, uuid = int(uuid), user_verified = False, admin_changed= True, partymember_changed= False, party_name= None,status= 'Pending',created_at= datetime.now(),edited_at= datetime.now())
+        session.add(role_update)
+        session.commit()
+        session.close()
+        return jsonify({'message': f'New admin role for user {uuid} is now waiting for verification.'}), 200
+
+    @staticmethod
+    def add_partymember_role(uuid, body):
+        session = Session()
+        role_update = RolesUpdatesDAO(id = int(uuid)*99999-2, uuid = int(uuid), user_verified= False, admin_changed= False, partymember_changed= True, party_name= body['party_name'], status = 'Pending', created_at= datetime.now(), edited_at= datetime.now())
+        session.add(role_update)
+        session.commit()
+        session.close()
+        return jsonify({'message': f'New party member role for user {uuid} is now waiting for verification.'}), 200
+
+    @staticmethod
+    def verify_role(uuid):
```

Comment:
This is not verifying role, this is checking if there is a new role. You can use this function (probably need to tweak it a little) to return this after logging in.